package.json

```diff
@@ -215,7 +220,7 @@
       "path": "./node_modules/cz-conventional-changelog"
     }
   },
-  "closureWhitelist": [
+  "typescriptWhitelist": [
```

Comment:
Is this even necessary? I find it odd that this works as-is without changing its contents, given that only mdc-base should actually contain typescript in this PR.

Reply:
closureWhitelist is something unique to MDC Web (not a closure thing). I believe this is for the `typescript-rewrite.sh` file.

Reply:
It seems like we will need this when we rewrite the other components, since it uses this list to know which ones to convert to relative paths for internal usages.

Reply:
We discussed this offline with the rest of the team and confirmed that closureWhitelist was only necessary for the closure compilation tests within this repository, to avoid trying to compile packages that hadn't yet been tested / updated for closure compilation. Thus it's safe to remove.